Makefile

```diff
@@ -32,7 +32,7 @@ install:
 	make deploy
 
 make_dirs:
-	mkdir -p ./logs && mkdir -p ./cafebabel/static/uploads/{articles,users,tags}
+	mkdir -p ./logs
```

Comment:
I couldn't figure where the folders are created now

Reply:
When we save we create the dir if it doesn't exist https://github.com/cafebabel/cafebabel.com/pull/200/files#diff-15bb7e77622f08f5bddbfa762b1df573R48

Reply:
Right. I guess it is less optimal to check the file path for every single save but it's more effective

Reply:
Agreed. We can optimize later.

Reply:
[Issue is open](https://github.com/cafebabel/cafebabel.com/issues/201)